packages/oasis-actions/src/strategies/ajna/earn/depositAndAdjust.ts

```diff
@@ -108,6 +112,26 @@ export async function depositAndAdjust(
     targetPosition = args.position.deposit(args.quoteAmount)
   }
 
+  if (!isPositionStaked && isDepositing && isReopening && shouldStakeNft) {
```

Comment:
Am I missing something or the only difference between those two conditions is second param in `encodeFunctionData`?
Then it could be shorter:
```ts
  if (!isPositionStaked && isDepositing && isReopening) {
    data = ajnaProxyActions.interface.encodeFunctionData(
      shouldStakeNft ? 'supplyQuoteMintNftAndStake' : 'supplyQuote',
        [
          args.poolAddress,
          ethers.utils.parseUnits(args.quoteAmount.toString(), args.quoteTokenPrecision).toString(),
          args.price.shiftedBy(18).toString(),
        ]
      )
    targetPosition = args.position.reopen(args.quoteAmount, priceToIndex)
  }
```

Reply:
good catch, fixed